### PR TITLE
MAINT: docs maintenance (WIP)

### PR DESCRIPTION
**Pull request by mmcky**
This PR

- [x] updates theme for `rtd`
- [x] adds a setup and install section
- [x] adds information for contributing to the package

some of this information is migrated from [organization website](quantecon.org)

### CODE REVIEW

**Comment by oyamad** [comment]
I don't know what it is happening with the [error with NumPy v2](https://github.com/QuantEcon/QuantEcon.py/actions/runs/11639934442/job/32416706795?pr=740). The testing with NumPy v2 was introduced by #736. Something may be wrong in there...

**Comment by oyamad** [comment]
The error should be resolved by #743.

**Comment by mmcky** [comment]
thanks @oyamad

File: docs/source/contributing.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+Write tests
+-----------
+
+All functions and methods contributed to QuantEcon.py should be paired with tests to verify that they are functioning correctly.
+
+Write documentation
+-------------------
+
+We try to maintain a simple and consistent format for inline documentation, known in the Python world as docstrings. 
+The format we use is known as :ref:`numpydoc<https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt>`.
```
Comment (oyamad) [inline docs/source/contributing.rst:26]: This page no longer exists.

**Comment by oyamad** [comment]
@mmcky Could you add an instruction about how to install with `conda`? There was once a confusion: see https://github.com/QuantEcon/QuantEcon.py/issues/732#issuecomment-2266373919.

File: docs/source/contributing.rst (hunk 1 above)
Comment (mmcky) [reply docs/source/contributing.rst]: thanks @oyamad migrating content at the moment from the QuantEcon org site and will proof read. Appreciate the comment.

**Comment by mmcky** [comment]
thanks @oyamad once this passes green I think we can merge.

File: docs/source/contributing.rst [hunk 2]
```diff
@@ -0,0 +1,49 @@
+Contribute to QuantEcon.py
+==========================
+
+If you would like to contribute to `QuantEcon.py <https://github.com/QuantEcon/QuantEcon.py>`_, 
+a good place to start is the `project issue tracker <http://github.com/QuantEcon/QuantEcon.py/issues>`_. 
+
+Set up a Conda development environment
+--------------------------------------
+
+One of the advantages of the `Anaconda Python environment <https://www.anaconda.com/distribution/>`_ is that it is
```
Comment (oyamad) [inline docs/source/contributing.rst:10]: Maybe this?
```suggestion
One of the advantages of the `Anaconda Python environment <https://www.anaconda.com/download>`_ is that it is
```

File: docs/source/contributing.rst [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+-----------
+
+All functions and methods contributed to QuantEcon.py should be paired with tests to verify that they are functioning correctly.
+
+Write documentation
+-------------------
+
+We try to maintain a simple and consistent format for inline documentation, known in the Python world as docstrings. 
+
+The format we use is known as `numpydoc <https://numpydoc.readthedocs.io/en/latest/format.html#overview>`_.
```
Comment (oyamad) [inline docs/source/contributing.rst:31]: Link to page top?
```suggestion
The format we use is known as `numpydoc <https://numpydoc.readthedocs.io/en/latest/format.html>`_. 
```

Comment (mmcky) [reply docs/source/contributing.rst:31]: 👍

**Comment by mmcky** [comment]
thanks for reviewing @oyamad -- merged all suggestions.

**Review by oyamad — APPROVED** [review]
@mmcky Thanks!

**Comment by mmcky** [comment]
- [x] readthedocs is now configured to give build previews for all PRs. 🥳

**Pull request merged**